Modix.Services/BehaviourConfiguration/BehaviourConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Modix.Services.BehaviourConfiguration
+{
+    public interface IBehaviourConfiguration
```

Comment:
Maybe `IBehaviourConfigurationManager`?

Subject to name changes of `BehaviorConfiguration`